packages/webdriverio/src/commands/browser/mock.ts

```diff
@@ -126,7 +127,7 @@ export default async function mock (url, filterOptions) {
     SESSION_MOCKS.add(networkInterception)
 
     if (this.isSauce) {
-        await networkInterception.init()
+        await (networkInterception as WebDriverNetworkInterception).init()
```

Comment:
you could probably do `const networkInterception: WebDriverNetworkInterception = new Net...` on line 126 and then you don't need the `as` here (or if not at least move the `as` there)